src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/OffenseTransformer.kt

```diff
@@ -17,6 +18,16 @@ class ConvictionTransformer {
     )
   } ?: emptyList()
 
+  fun transformToApi(caseDetail: CaseDetail) = caseDetail.offences.map {
+    ActiveOffence(
+      deliusEventNumber = it.eventNumber,
+      offenceDescription = it.description,
+      offenceId = it.id.toString(),
+      convictionId = it.eventId,
+      offenceDate = it.date,
```

Comment:
can you please let me know once this gets to pre-prod, i'd like to do some 'like for like' comparison when creating an application for a CRN that already has an application against it that was using the 'old' endpoint

Reply:
will do!